ritasamir/assignment1/Q_01.java

```diff
@@ -0,0 +1,44 @@
+package assignment_1;
+
+import java.util.Arrays;
+
+public class Q_01 {
+
+	/**
+	 * checkAnagram is a method that takes two sentences and checks if they are
+	 * anagram or not 
+	 * @param firstStr is the first sentence
+	 * @param secondStr is the second sentence
+	 * @return true if they are anagram, false otherwise.
+	 */
+	public boolean checkAnagram(String firstStr, String secondStr) {
+		String[] array1 = firstStr.split(" ");
+		String[] array2 = secondStr.split(" ");

```

Comment:
optional: I'd remove the splitter so that sentences like "hello world" & "hell oworld" or "hello world" & "world hello" are acceptable anagrams.